pkg/reconciler/v1alpha1/configuration/resources/revision.go

```diff
@@ -18,11 +18,21 @@ package resources
 
 import (
 	"fmt"
+	"strconv"
+	"time"
 
 	"github.com/knative/serving/pkg/apis/serving"
 	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
 	"github.com/knative/serving/pkg/reconciler"
 	"github.com/knative/serving/pkg/reconciler/v1alpha1/configuration/resources/names"
+	"github.com/knative/serving/pkg/system"
+)
+
+const (
+	// Delay duration after a revision create before considering it for GC
+	staleRevisionCreateDelay = 5 * time.Minute
+	// Timeout since a revision lastPinned before it should be GC'd
+	staleRevisionTimeout = 5 * time.Minute
```

Comment:
Can these go into `config-controller.yaml` (with sensible defaults)?

Reply:
I agree these should, but this is going to be a bit of work I think. It seems config-controller.yaml is really 'config-route-controller.yaml' as the route controller is the only one which parses / stores this config. Im wondering if I should try promoting `pkg/reconciler/v1alpha1/route/config` to be global across controllers or whether we should split up configmaps?

Reply:
Ah, `config-controller` is actually *revision* only, hmm.  As much as I hate to suggest it, maybe `config-gc.yaml` is the best short-term option.  cc @dprotaso has been in here most recently and may have thoughts.